### DISCUSSION TITLE
any good openapi renderers recommendations?

### DISCUSSION THREAD

**Discussion by grmkris** [wevm/vocs#27] [General]
sorry for using issues as questions, if you don't want this I can drop it haha, but anyway i'd also like to render openapi swagger docs in my project

https://www.npmjs.com/package/swagger-ui-react was able incorprate this quite easily, but it doesn't look nice and would require me to build a custom theme for it.

do you suggest some other library for this? probably would be a nice plugin to have as well

**Comment by tmm** [comment]
Thanks for sharing! Still early days for Vocs and we have a shortlist of features already on the roadmap. Will revisit this later! (In the meantime, feel free to use this discussion thread as a place to share any more thoughts on this topic.)

**Comment by taylorjdawson** [comment]
@grmkris I'd be interested in seeing what solution you came to!

**Comment by huangjunjia** [comment]
I think scalar may be a more suitable solution. https://guides.scalar.com/scalar/scalar-api-references/scalar-api-reference-for-react

![CleanShot 2024-10-31 at 01 06 16@2x](https://github.com/user-attachments/assets/7dea3531-3a5f-430e-9890-3a08fdcc5092)

**Reply by grmkris** [reply]
yes today i'd go with scalar instead of swagger

**Comment by yinxulai** [comment]
This feature seems to be getting more and more important.
